src/axios/axiosInstance.ts

```diff
@@ -18,18 +18,11 @@ const endpointsWithoutToken = [ENDPOINTS.signin, ENDPOINTS.signup];
 axiosInstance.interceptors.request.use(
   async (config: InternalAxiosRequestConfig) => {
     const accessToken = await SecureStore.getItemAsync(SecureStorageKey.ACCESS_TOKEN);
-    const refreshToken = await SecureStore.getItemAsync(SecureStorageKey.ACCESS_TOKEN);
 
     if (config.url && accessToken && !endpointsWithoutToken.includes(config.url)) {
       config.headers.Authorization = `Bearer ${accessToken}`;
     }
 
-    if (config.url === ENDPOINTS.signout && refreshToken) {
```

Comment:
this kind of changes should be merged in another pr/fix

Reply:
yeap, got it)